make.ps1

```diff
@@ -1,27 +1,5 @@
-function FindMSBuild
-{
-	$msBuildVersions = @("4.0")
-	foreach ($msBuildVersion in $msBuildVersions)
-	{
-		$key = "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\{0}" -f $msBuildVersion
-		$property = Get-ItemProperty $key -ErrorAction SilentlyContinue
-		if ($property -eq $null -or $property.MSBuildToolsPath -eq $null)
-		{
-			continue
-		}
-		$path = Join-Path $property.MSBuildToolsPath -ChildPath "MSBuild.exe"
-		if (Test-Path $path)
-		{
-			return $path
-		}
-	}
-	return $null
-}
-
-function UtilityNotFound
-{
-	echo "OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command."
-}
+$functions = "$($MyInvocation.MyCommand.path | split-path)\make-functions.ps1" 
```

Comment:
You will get most of the benefit, but it would take a few more seconds figuring out that the heart of the code is at the bottom (unless this is some kind of an agreed practice in powershell i am simply unaware of)

Basically I tried making the file as accessible as possible for someone opening it for the first time.


Though if there's strong agreement that its better not to have 2 files I don't mind combining them.